### PR TITLE
Updated Home Page

### PR DESCRIPTION
**Pull request by IanSaucy**

### CODE REVIEW

**Comment by Aykerimm** [comment]
Sure ,no problem 
> @Aykerimm Can you put the "What we do" below the iframe embed?

File: src/pages/MasterPage.tsx [hunk 1]
```diff
@@ -38,7 +37,6 @@ export const Main: React.FC = () => {
          */}
         <Routes>
           <Route path={InternalRoutes.HOME.path} element={<LandingPage />}></Route>
-          <Route path={InternalRoutes.TRACKER.path} element={<TrackerPage />}></Route>
           <Route path={InternalRoutes.PUBLICATIONS.path} element={<PublicationsPage />}></Route>
           <Route path={InternalRoutes.TEAM.path} element={<OurTeamPage />}></Route>
           <Route path={InternalRoutes.RESOURCES.path} element={<ResourcesPage />}></Route>
```
Comment (toucham) [inline src/pages/MasterPage.tsx:42]: same as suggested above:
`<Route ... />`

**Review by toucham — CHANGES_REQUESTED** [review]
small changes

**Comment by Aykerimm** [comment]
I am trying to merge the changes from react_transfer branch. Should I change the death tracker app from RshinyApp to Tableau to be on the main page?

**Pull request merged**